QT_Py_Timer/QT_Py_Hydration_Reminder.py

```diff
@@ -0,0 +1,189 @@
+import time
+import board
+import adafruit_lis3dh
+import neopixel
+
+# The interval you would like to be reminded to drink water, in minutes.
+hydration_reminder = 60
+
+# Length of time, in minutes, before timer begins Red Alert blinking.
+red_alert_delay = 2
+
+# The color will fade from color_begin to color_complete. This is a gentle indicator of how much
+# time has passed. Set to any two colors.
+color_begin = (0, 0, 255)  # Blue. Set to any color to begin your countdown.
+color_complete = (255, 255, 255)  # White. Set to any color to end your countdown.
+led_brightness = 0.2  # Set to a number between 0.0 and 1.0 to set LED brightness.
+
+# Increase or decrease this to change the speed of the Red Alert blinking in seconds.
+red_alert_blink_speed = 0.5
+
+# These are the thresholds the z-axis values must exceed for the orientation to be considered
+# "up" or "down". These values are valid if the LIS3DH breakout is mounted horizontally inside the
+# timer assembly. If you want the option to have the timer on an angle while timing, you can
+# calibrate these values by uncommenting the print(z) in orientation() to see the z-axis values and
+# update the thresholds to match the desired range.
+down_threshold = -8
+up_threshold = 8
+
+# Number of LEDs. Default is 32 (2 x rings of 16 each). If you use a different form of NeoPixels,
+# update this to match the total number of pixels.
+number_of_pixels = 32
+
+# Set up accelerometer and LEDs.
+lis3dh = adafruit_lis3dh.LIS3DH_I2C(board.I2C())
+pixels = neopixel.NeoPixel(board.A3, number_of_pixels, brightness=led_brightness)
+pixels.fill(0)  # Turn off the LEDs if they're on.
+
+STATE_IDLE = "Idle"
+STATE_TIMING = "Timing"
+STATE_TIMING_COMPLETE = "Timing complete"
+RED_ALERT = "Red Alert"
+
+
+def minutes_to_seconds(minutes):
+    """
+    Convert minutes to seconds for use with time.monotonic().
+
+    :param minutes: The number of minutes. Can be a float, e.g. 0.5 for half a minute.
+    """
+    seconds = minutes * 60
+    return seconds
+
+
+def gradient(color_one, color_two, blend_weight):
```

Comment:
fancyled has a function like this if you can include that library

Reply:
Good to know. However, the current libs barely fit, another one would not.